### DISCUSSION TITLE
i have changed my username but when i am searching google its showing old one after pressing enter its coming 404 not found

### DISCUSSION THREAD

**Discussion by Syeda-Somiya-Tasnim** [community/community#121814] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i have changed my username but when i am searching google its showing old one after pressing enter its coming 404 not found

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Syeda-Somiya-Tasnim ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by YassineCherkaoui** [comment]
hey @Syeda-Somiya-Tasnim,
It seems like Google's search index hasn't updated to reflect your new username yet. This can happen occasionally, as search engines don't instantly update their indexes. Typically, search engines crawl and update their indexes periodically, but it can take some time for changes to be reflected.

If you've recently changed your username, it's possible that Google hasn't crawled your GitHub profile since the change occurred. However, once Google does crawl your profile again, it should update the search results to reflect your new username.

In the meantime, if you're encountering a 404 error when clicking on the search result for your GitHub profile, it could be because the old username is still present in the URL. Since your profile's URL is likely based on your username, changing your username may result in the old URL becoming invalid.

You can try updating any links or bookmarks you have to your GitHub profile with your new username to avoid this issue. Additionally, you may want to reach out to Google's support or wait for their search index to update naturally.

**Comment by davevad93** [comment]
Hi @Syeda-Somiya-Tasnim 

A couple of months ago I've changed my GitHub username and profile description as well, the first couple of days when searching my profile via Google it kept showing my old one with the old description. I don't know exactly the reason I guess that the search engines need a little of time to update everything, so I'd tell you to wait 2-3 days. Afterwards you'll surely be able to find your new username on Google.

**Reply by ebndev** [reply]
I second this!
